include/rocksdb/utilities/secondary_index_faiss.h

```diff
@@ -29,4 +30,24 @@ namespace ROCKSDB_NAMESPACE {
 std::unique_ptr<SecondaryIndex> NewFaissIVFIndex(
     std::unique_ptr<faiss::IndexIVF>&& index, std::string primary_column_name);
 
+// Helper methods to convert embeddings from a span of floats to Slice or vice
+// versa
+
+// Convert the given span of floats of size dim to a Slice.
+// PRE: embedding points to a contiguous span of floats of size dim
+inline Slice AsSlice(const float* embedding, size_t dim) {
```

Comment:
These are in rocksdb namespace, should we have more descriptive names for them?

Reply:
Sure, how about something like `ConvertFloatsToSlice` and `ConvertSliceToFloats` ?

Reply:
Sounds great, thanks!